src/plugins/intel_gpu/src/kernel_selector/cl_kernels/prior_box_ref.cl

```diff
@@ -121,52 +121,56 @@ KERNEL(ref)
         #endif
     }
 
-    for (uint ms_idx = 0; ms_idx < MIN_SIZE_SIZE; ++ms_idx) {
-        box_width = MIN_SIZE[ms_idx] * 0.5f;
-        box_height = MIN_SIZE[ms_idx] * 0.5f;
-        FUNC_CALL(calculate_data)(center_x, center_y, box_width, box_height, false, &out_index, output);
-        #ifdef MIN_MAX_ASPECT_RATIO_ORDER
-            if (MAX_SIZE_SIZE > ms_idx) {
-                box_width = box_height = sqrt(MIN_SIZE[ms_idx] * MAX_SIZE[ms_idx]) * 0.5f;
-                FUNC_CALL(calculate_data)(center_x, center_y, box_width, box_height, false, &out_index, output);
+    // Explicitly check MIN_SIZE has value to avoid seg fault duing opencl build
+    if (MIN_SIZE_SIZE > 0) {
```

Comment:
This condition check is unnecessary. It is checked by the next `for` statement.

Reply:
Why those are blank? I think it can always default to 0, isn't it?

Reply:
When `MAX_SIZE_SIZE` is 0, `MAX_SIZE` is defined as `#define MAX_SIZE (float []){  }`
In this case, `MAX_SIZE[ms_idx]` causes programs to crash.
But, if this code is under a condition `MAX_SIZE_SIZE > 0`, it looks like that this code is excluded before compilation.